code/modules/mob/living/brain/brain_item.dm

```diff
@@ -218,6 +218,11 @@
 	desc = "We barely understand the brains of terrestial animals. Who knows what we may find in the brain of such an advanced species?"
 	icon_state = "brain-x"
 
+/obj/item/organ/brain/positron
+	name = "Positronic Brain"
+	desc = "A positronic brain used in production of silicon based lifeforms"
```

Comment:
`.`

Reply:
this is because MMI code is spaghetti, and ill need to completely rewrite it to use the original posi brains. Also, original positronic brains cannot be used as organs

Reply:
Also, `silicon-based`

Reply:
I meant the lack of a period you fool.